cmd/ftl/cmd_build.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+
+	"github.com/alecthomas/errors"
+
+	"github.com/TBD54566975/ftl/backend/common/exec"
+	"github.com/TBD54566975/ftl/backend/common/log"
+	"github.com/TBD54566975/ftl/backend/common/moduleconfig"
+)
+
+type buildCmd struct {
+	Base string `arg:"" help:"Directory containing ftl.toml" type:"existingdir" default:"."`
+}
+
+func (b *buildCmd) Run(ctx context.Context) error {
+	// Load the TOML file.
+	config, err := moduleconfig.LoadConfig(b.Base)
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	switch config.Language {
+	case "kotlin":
+		return b.buildKotlin(ctx, config)
+	default:
+		return errors.Errorf("unable to build. unknown language %q", config.Language)
+	}
+}
+
+func (b *buildCmd) buildKotlin(ctx context.Context, config moduleconfig.ModuleConfig) error {
+	logger := log.FromContext(ctx)
+
+	buildCmd := config.Build
+	if buildCmd == "" {
+		buildCmd = "source ../bin/activate-hermit && mvn compile"
```

Comment:
You'll have to check if the `../bin/hermit` directory exists, as it's optional.

Reply:
Actually I would remove the Hermit activation completely. We should assume that the user has either already activated the environment, or isn't using it.

Reply:
Yeah I was curious about that. In our "default" build would we assume it's there? Or, should I just always check if there is one and if so go ahead and add the activation?